pillar/nginx/ocw_origin.sls

```diff
@@ -65,4 +65,6 @@ nginx:
                         - $uri/
                         - index.htm
                         - index.html
-                        - =404
```

Comment:
This line doesn't need to be removed, that will just direct Nginx to return a 404 response when no files are found. The config that you added is what will handle determining what gets loaded for that response.

Reply:
OK. Commit 0204d85 puts it back.